src/current/v24.3/manage-logical-data-replication.md

```diff
@@ -0,0 +1,199 @@
+{{site.data.alerts.end}}
+
+To manage the DLQ, you can evaluate entries in the `incoming_row` column and apply the row manually to another table with SQL statements.
+
+As an example, for an LDR stream created on the `movr.public.promo_codes` table:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW TABLES;
+~~~
+
+The table will have a random number within its name to ensure it is unique.
+
+~~~
+    schema_name    |         table_name         | type  | owner | estimated_row_count | locality
+-------------------+----------------------------+-------+-------+---------------------+-----------
+  crdb_replication | dlq_113_public_promo_codes | table | node  |                 186 | NULL
+  public           | promo_codes                | table | root  |                1047 | NULL
+  public           | rides                      | table | root  |                 976 | NULL
+  public           | user_promo_codes           | table | root  |                 134 | NULL
+  public           | users                      | table | root  |                 424 | NULL
+  public           | vehicle_location_histories | table | root  |               13012 | NULL
+  public           | vehicles                   | table | root  |                 153 | NULL
+(7 rows)
+~~~
+
+The schema for the `movr.crdb_replication.dlq_113_public_promo_codes` DLQ:
+
+~~~sql
+CREATE TABLE crdb_replication.dlq_113_public_promo_codes (
+    id INT8 NOT NULL DEFAULT unique_rowid(),
+    ingestion_job_id INT8 NOT NULL,
+    table_id INT8 NOT NULL,
+    dlq_timestamp TIMESTAMPTZ NOT NULL DEFAULT now():::TIMESTAMPTZ,
+    dlq_reason STRING NOT NULL,
+    mutation_type crdb_replication.mutation_type NULL,
+    key_value_bytes BYTES NOT VISIBLE NOT NULL,
+    incoming_row JSONB NULL,
+    crdb_internal_dlq_timestamp_id_ingestion_job_id_shard_16 INT8 NOT VISIBLE NOT NULL AS (mod(fnv32(md5(crdb_internal.datums_to_bytes(dlq_timestamp, id, ingestion_job_id))), 16:::INT8)) VIRTUAL,
+CONSTRAINT dlq_113_public_promo_codes_pkey PRIMARY KEY (ingestion_job_id ASC, dlq_timestamp ASC, id ASC) USING HASH WITH (bucket_count=16)
+)
+~~~
+
+## Schema changes
+
+When you start LDR on a table, the job will lock the schema, which will prevent any accidental [schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}) that would cause issues for LDR. There are some [supported schema changes](#supported-schema-changes) that you can perform on a replicating table, otherwise it is necessary to stop LDR in order to [coordinate the schema change](#coordinate-other-schema-changes).
+
+### Supported schema changes
+
+There are some supported schema changes, which you can perform during LDR **without** restarting the job:
+
+Allowlist schema change | Exceptions
+-------------------+-----------
+[`CREATE INDEX`]({% link {{ page.version.version }}/create-index.md %}) | <ul><li>Hash-sharded indexes</li><li>Indexes with a computed column</li><li>Partial indexes</li></ul>
```

Comment:
unique indexes cannot be created mid ldr stream.

Reply:
added